docs/design/hub-and-spoke/switch-validate/switch-validate.md

```diff
@@ -5,9 +5,667 @@
+
+You can combine several flags in one query e.g. to get full info only about groups and meters use query
+params `groups` and `meters`.
+
+`GET https://{host}/v2/switches/{switch_id}/validate?include=groups|meters`
+
+### Excluding flow info from response
+
+Request params
+
+To exclude flow info from response use `flow_info`.
+
+`GET https://{host}/v2/switches/{switch_id}/validate?exclude=flow_info`
+
+<details><summary>Response payload</summary>
+
+```json
+{
+  "as_expected": true,
+  "groups": {
+    "as_expected": true,
+    "excess": [
+      <v2GroupInfo> // without flow_id and flow_path
+    ],
+    "missing": [
+      <v2GroupInfo> // without flow_id and flow_path
+    ],
+    "proper": [
+      <v2GroupInfo> // without flow_id and flow_path
+    ],
+    "misconfigured": [
+      {
+        "id": "12123213",
+        "expected": {
+          <v2GroupInfo>  // without flow_id and flow_path
+        },
+        "discrepancies": {
+          <v2GroupInfo>  // without flow_id and flow_path
+        }
+      }
+    ]
+  },
+  "logical_ports": {
+    "as_expected": true,
+    "excess": [
+      <v2LogicalPortInfo>
+    ],
+    "missing": [
+      <v2LogicalPortInfo>
+    ],
+    "proper": [
+      <v2LogicalPortInfo>
+    ],
+    "misconfigured": [
+      {
+        "id": "12123213",
+        "expected": {
+          <v2LogicalPortInfo>  
+        },
+        "discrepancies": {
+          <v2LogicalPortInfo>  
+        }
+      }
+    ]
+  },
+  "meters": {
+    "as_expected": true,
+    "excess": [
+      <v2MeterInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "missing": [
+      <v2MeterInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "proper": [
+      <v2MeterInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "misconfigured": [
+      {
+        "id": "12123213",
+        "expected": {
+          <v2MeterInfo>  // without flow_id, y_flow_id and flow_path
+        },
+        "discrepancies": {
+          <v2MeterInfo>  // without flow_id, y_flow_id and flow_path
+        }
+      }
+    ]
+  },
+  "rules": {
+    "as_expected": true,
+    "excess": [
+      <v2RuleInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "missing": [
+      <v2RuleInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "proper": [
+      <v2RuleInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "misconfigured": [
+      {
+        "id": "12123213",
+        "expected": {
+          <v2RuleInfo>  // without flow_id, y_flow_id and flow_path
+        },
+        "discrepancies": {
+          <v2RuleInfo>  // without flow_id, y_flow_id and flow_path
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+### Including/excluding info in response
+
+To exclude flow info from response use `exclude=flow_info`. You can combine it with `include=groups|logical_ports|meters|rules` 
+in query e.g.
+
+Ex 1. `GET https://{host}/v2/switches/{switch_id}/validate?exclude=flow_info&include=groups|meters`
```

Comment:
What will be behaviour if field included into "include" and into "exclude" lists?